templates/download/server/download.html

```diff
@@ -0,0 +1,261 @@
+{% extends "download/_base_download.html" %}
+
+{% block title %}Thank you for downloading Ubuntu Server{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1S5zY7HN_Q8eQWch0mCEfLICfj8pW1bct9_wGArCNIqc/edit{% endblock meta_copydoc %}
+
+{% block body_class %}is-paper{% endblock body_class %}
+
+{% block head_extra %}
+  <meta name="robots" content="noindex" />
+{% endblock %}
+
+{% block content %}
+<noscript>
+  <meta
+    http-equiv="refresh"
+    content="3;url=https://releases.ubuntu.com/{{ version }}/ubuntu-{{ version }}-live-server-amd64.iso"
+  />
+</noscript>
+
+<section class="p-strip is-shallow u-no-padding--bottom">
+  <div class="row--50-50">
+    <div class="col">
+      <h1 class="js-download-option">Thank you for downloading Ubuntu Server {{ version }}</h1>
+      <p>Your download should start in the background. If it doesn't,
+        <a href="https://releases.ubuntu.com/{{ version }}/ubuntu-{{ version }}-live-server-amd64.iso">download&nbsp;now</a>.<br />
+        {% with version=version, system="live-server", architecture="amd64" %}
+          {% include "download/shared/_verify-checksums.html" %}
+        {% endwith %}
+      </p>
+    </div>
+    <div class="col">
+      {% include 'shared/_server-download-newsletter.html' %}
+    </div>
+  </div>
+</section>
+
+{% include 'download/shared/_server_weekly_news.html' %}
+
+<section class="p-section">
+  <hr class="p-rule is-fixed-width" />
+  <div class="row--50-50">
+    <div class="col">
+      <h2>Ubuntu CLI cheatsheet</h2>
+      <div class="p-section--shallow">
+        <p>Learn how to use the command line efficiently and get started with DevOps &mdash; from basic file management to LXD virtualisation and Ubuntu Pro.</p>
+      </div>
+      <button class="p-button--positive" onclick="toggleModal()">Download the CLI cheat sheet</button>
```

Comment:
why does this need a separate onclick? Was it not possible to use 'js-invoke-modal' here?

Reply:
I'm not sure, this page was not in scope for this ticket. For some reason it was added to the pr as a new page after I rebased but I did not work on this